akka-docs/src/main/paradox/cluster-client.md

```diff
@@ -3,7 +3,7 @@
 An actor system that is not part of the cluster can communicate with actors
 somewhere in the cluster via this @unidoc[ClusterClient]. The client can of course be part of
 another cluster. It only needs to know the location of one (or more) nodes to use as initial
-contact points. It will establish a connection to a @unidocRef[ClusterReceptionist](akka.cluster.client.ClusterReceptionist) somewhere in
+contact points. It will establish a connection to a @unidoc[client.ClusterReceptionist] somewhere in
```

Comment:
This is a bit strange though, it's neither the FQCN nor the simple name but something in-between, I think either `@unidoc[ClusterReceptionist]` or `@unidoc[akka.cluster.client.ClusterReceptionist]` is what we'd want

Reply:
OK, added a new commit to only allow FQCN or a simple class name (without any `.`).